openapi/platform/platform.yaml

```diff
@@ -2908,6 +2908,10 @@ paths:
         name: 'filter[partner.id]'
         in: query
         required: true
+      - schema:
+          type: string
+        name: 'filter[businessLocation.id]'
+        in: query
```

Comment:
Apparently the `filter[partner.id]` parameter was added both at the path level (here) and at the method/operation level (50 lines down). Based on Vendasta's usage, query params should only exist at the operation level since they typically don't apply to all methods. 

Please also add a description. We should call out that a partner.id filter is always required while businessLocation.id is an optional filter within the partner.